### DISCUSSION TITLE
Support ARM64/aarch64

### DISCUSSION THREAD

**Discussion by bgrainger** [microsoft/azure-container-apps#1363] [Ideas]
Per https://learn.microsoft.com/en-us/azure/container-apps/containers:

> Azure Container Apps supports:
> * Any Linux-based x86-64 (`linux/amd64`) container image

I'd like support for ARM64 (`aarch64`) on Linux. Many runtimes, including .NET 8 (and later), support running on ARM64.

I would expect a corresponding decrease in vCPU-second pricing for Azure Container Apps on the Consumption Plan based on the lower cost of this hardware (ideally around a 37% decrease):

* D2d v5 (3rd Generation Intel® Xeon® Platinum 8370C) in East US is $0.1130/hour.
* D2ps v5 (Arm architecture) is East US is $0.0770/hour
* D2ps v6 (Azure’s first-generation Cobalt 100 processor) in East US is $0.0702/hour

I would also expect a lower cost for provisioning a dedicated workload profile (but support in the Consumption Plan would be sufficient for my immediate needs).

**Comment by stunney** [comment]
Duplicate; welcome to our He🏒🏒

https://github.com/microsoft/azure-container-apps/issues/569
